pkgs/data/fonts/powerline-symbols/default.nix

```diff
@@ -5,7 +5,7 @@ let
 in runCommand "powerline-symbols-${version}" {
   meta = {
     inherit (powerline.meta) license;
-    priority = (powerline.meta.priority or 0) + 1;
+    priority = (powerline.meta.priority or lib.meta.defaultPriority) + 1;
```

Comment:
this priority is now 6 instead of 1 (meaningful!)

Reply:
Okay, so it looks like this is the one change that might break stuff. Do we think this was just meant to be `- 1`, or is it intentional and we’re fixing it?

Reply:
I think it's meant to make the powerline symbols not cause a conflict with the Python powerline application, from which they're copied.

They're the same:

$ nix-build --no-out-link -A powerline -A powerline-symbols | xargs fd -tf 'otf|conf' | xargs sha1sum | sort | rg -A1 -- -symbols-2.8.3
15dc17df4158eef4e0605d3637b903269aa61754  /nix/store/23alb0rgiixp6n4s2givj5406j7xv2fq-powerline-symbols-2.8.3/etc/fonts/conf.d/10-powerline-symbols.conf
15dc17df4158eef4e0605d3637b903269aa61754  /nix/store/2dcvqi45pdi191y5rkqa1igz6988xjkc-python3.12-powerline-2.8.3/etc/fonts/conf.d/10-powerline-symbols.conf
--
b5eff3c621132f75195c2cef18ba06d65547646b  /nix/store/23alb0rgiixp6n4s2givj5406j7xv2fq-powerline-symbols-2.8.3/share/fonts/OTF/PowerlineSymbols.otf
b5eff3c621132f75195c2cef18ba06d65547646b  /nix/store/2dcvqi45pdi191y5rkqa1igz6988xjkc-python3.12-powerline-2.8.3/share/fonts/OTF/PowerlineSymbols.otf

Reply:
SGTM.